sdks/python/hatchet_sdk/worker/runner/runner.py

```diff
@@ -419,23 +465,18 @@ async def handle_cancel_action(self, action: Action) -> None:
         try:
             # call cancel to signal the context to stop
             if key in self.contexts:
-                context = self.contexts.get(key)
-
-                if context:
-                    context._set_cancellation_flag()
+                self.contexts[key]._set_cancellation_flag()
```

Comment:
drive by cleanup - the `.get` and then checking for truthiness in these is redundant b/c if the key is in the set of keys, then we know the value is truthy already